packages/DynamicHyperlink/src/DynamicHyperlink.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import "./DynamicHyperlink.scss";
+
+export default function DynamicHyperlink({ onFetch }) {
+  const I18n = useI18n();
+
+  React.useEffect(() => {
+    function updateDynamicHyperlinks() {
+      const dynamicHyperlinks = Array.from(document.querySelectorAll("[data-dynamic-hyperlink]"));
+
+      dynamicHyperlinks
+        .filter(dynamicHyperlink => !dynamicHyperlink.hasAttribute("data-dynamic-hyperlink--processed"))
+        .forEach(dynamicHyperlink => {
+          const originalLinkUrl = dynamicHyperlink.innerHTML;
```

Comment:
i think we should use the href value

Reply:
it's the same value, is it not?

Reply:
It is right now.  But it is possible that in the future we store transformed links in the db and the frontend will refetch.  

Another reason to use the Href attribute is that is what the dyn-hyperlinking package in Projects is using - so a change here will make the this component consistent.  Maybe we can chat about this during the meeting today.